### ISSUE TITLE
BUG: media:clearthumbnails command misses a newline in its output

### ISSUE THREAD

**Issue by mrimann** [neos/neos-development-collection#3894]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The output of the `media:clearthumbnails` command is missing a new line after it finishes (e.g. when the progress bar shows 100%).

Example output, anonymized:
```
user@webserver:~/httpdocs/releases/current$ FLOW_CONTEXT=Production ./flow media:clearthumbnails
 189/189 [============================] 100%user@webserver:~/httpdocs/releases/current$
```

This "breaks" the the visual output and the shell's prompt (`user@webserver(...)` in the above example is shown on the same line as the progress bar. It just looks ugly and disturbs me every time I use this command.

### Expected Behavior

IMO it would be better, if after ending the progress bar (when it reaches 100%), a line break would be emitted, so that the output would look more like this made up output:
```
user@webserver:~/httpdocs/releases/current$ FLOW_CONTEXT=Production ./flow media:clearthumbnails
 189/189 [============================] 100%
user@webserver:~/httpdocs/releases/current$
```

The change is just that there's now a line-break between `100%` and the `user@` text.

### Steps To Reproduce

Just run the command in a working Neos setup and look at the output.

### Environment

```markdown
- Flow: probably all of them
- Neos: 4.x, 5.x, 7.x
- PHP: different, those matching to the Neos version
```

**Comment by mrimann** [comment]
I think one could just add an `echo(PHP_EOL)` or similar to the this line https://github.com/neos/neos-development-collection/blob/master/Neos.Media/Classes/Command/MediaCommandController.php#L356 - but I'm not sure if that's a clean solution.

**Comment by crydotsnake** [comment]
Hello @mrimann 

Thanks a lot for reporting this! Unfortunately i could not reproduce this locally with Neos 7, and 8. Maybe it is something with your shell on your server? Could you try it also locally on a Neos Installation?

**Comment by mrimann** [comment]
Hi @crydotsnake 

That's strange - I've never seen it "work", only ever the variant I showed in this bug report / request for improvement. The setup I've seen it most was on Ubuntu servers with a regular bash as the user's shell.

I've no local setup directly on my computer, but instead we use ddev for our development work. So I tested it there (with a Neos 5.x Setup):
```
mario.rimann@projectxyz-web:/var/www/html$ ./flow media:clearthumbnails
    0 [->--------------------------]mario.rimann@projectxyz-web:/var/www/html$
```

So the output is also the same way as on our production systems.

Then I've cloned the [neos-base-distribution](https://github.com/neos/neos-base-distribution) + started that one up in ddev (e.g. `ddev config`, then changed the PHP-Version to 8.1 in `.ddev/config.yaml` file followed by `ddev composer install` and then via `ddev ssh` within the container the following steps):

```
# to warm up the caches initialla
mario.rimann@neos-vanilla-web:/var/www/html$ ./flow help

# create DB schema (otherwise clearthumbnails command will fail anyway)
mario.rimann@neos-vanilla-web:/var/www/html$ ./flow doctrine:migrate

# try to remove thumbnails (as expected there are none!)
mario.rimann@neos-vanilla-web:/var/www/html$ ./flow media:clearthumbnails
    0 [->--------------------------]mario.rimann@neos-vanilla-web:/var/www/html$
```

**So it's reproducable for me with a bash on different setups** (remote prod server + locally in ddev).

If I try the same command again **within a zsh session** (within the ddev container after installing zsh with `sudo apt-get update && sudo apt-get install zsh && zsh`), **the output differs and looks as it should look like IMO:**
```
neos-vanilla-web% ./flow media:clearthumbnails
    0 [->--------------------------]%
neos-vanilla-web%
```

So I think it's dependent on the Shell used by the user. But since bash is like "the default" for many Linux distributions as far as I understand, I think it still makes sense to fix this.

**Comment by crydotsnake** [comment]
Hello @mrimann !

Thank you very much for your detailed answer! I was able to reproduce the problem by changing my shell on my local system to bash.

**Comment by tantegerda1** [comment]
Affected commands in Neos.Media:
- `media:clearthumbnails`
- `media:createthumbnails`
- `media:removeunused`

**Comment by tantegerda1** [comment]
As mentioned before, various shells treat output without a final newline differently, sometimes [based on configuration](https://zsh.sourceforge.io/Doc/Release/Options.html#Prompting).

By default, `zsh` "recovers" from an output without a newline:
```
user $ echo -n foo
foo%       <-- the percent sign hints at something being fishy
user $
```

`bash` however, does not:
```
user $ echo -n foo
foouser $
     ^-- next prompt right after the output
```

**Comment by tantegerda1** [comment]
See also: https://github.com/symfony/symfony/issues/21590

**Comment by crydotsnake** [comment]
Fixed with: #3902

**Comment by mrimann** [comment]
Thanks @crydotsnake and @tantegerda1 for fixing this! Looking forward to seeing this in action after the next patch-level updates :-)
